src/pages/blog/starlingx-subcloud-local-install.md

```diff
@@ -0,0 +1,65 @@
+---
+templateKey: blog-post
+title: StarlingX Subcloud Local Install Enhancements
+author: Shrikumar Sharma
+date: 2022-04-11T14:26:09.627Z
+category:
+  - label: Features & Updates
+    id: category-A7fnZYrE1
+---
+
+Learn more about the new feature, Subcloud Local Install enhancements, in StarlingX, below.
+StarlingX Distributed Cloud system supports a scalable number of geographically distributed StarlingX edge systems, centrally managed and synchronized over L3 networks from a StarlingX central region. Each edge system is also highly scalable, from a single StarlingX Simplex deployment to a full Standard Cloud configuration with storage nodes.
+
+[StarlingX Cloud Deployment](/img/StarlingX_CloudDeployment.png)
+
+As illustrated in Figure 1, the network is distributed over a vast area. Each region is a geographical grouping, or an area on a map. The edge clouds provide the Edge Computing power closer to the users of the cloud services, located in cities and try to decrease latency. Observe that there are network delays and losses over all links in the deployment.
+This naturally brings up the question of deployment of the clouds. We need to deploy as many edge clouds as possible while keeping in mind that data will be transmitted over possibly low-bandwidth, unreliable and unpredictable networks. 
+
+A typical deployment will involve the transfer of a few gigabytes of data, which will be used in the installation process. Transmitting a few gigabytes over a possibly low-bandwidth network will consume a significant amount of time.
+A typical edge cloud deployment may consist of multiple controllers and potentially compute and storage nodes, connected over a network that is designed per the user’s specifications. This implicitly means that there are potential multiple points of delay and latency for network traffic.
```

Comment:
```suggestion
A typical edge cloud deployment may consist of multiple controllers and potentially compute and storage nodes, connected over a network that is designed for the user’s specifications. This can lead to a complex environment to build out with multiple failure points.
```